README.md

```diff
@@ -132,7 +132,7 @@ When you want to make changes to the package to use in another repository, it ca
 
 - Open the file you wish to change in the conda environment
 - Do this by clicking into functions from the package in your IDE or by finding the file you wish to change in your file system, e.g.
- ```.../anaconda3/envs/local_government_dashboard/lib/python3.9/site-packages/gov_uk_dashboards/components/...```
+ ```.../anaconda3/envs/*repo*/lib/python3.9/site-packages/gov_uk_dashboards/components/...```
```

Comment:
It seems really hard to find the file you want this way I think. Does anyone still access via anaconda?

Reply:
Yeah, definitely better to click into functions from the package, but this was there as a backup if that wasn't available